components/family-studies/api/src/main/java/org/phenotips/studies/family/internal/export/FamilySearchResult.java

```diff
@@ -71,6 +73,21 @@ public FamilySearchResult(Patient patient, boolean usePatientName, Family family
 
     }
 
+    @Override
+    public boolean equals(Object other)
```

Comment:
You always need to override equals & hashCode for any class that is used in a Set or as key in a Map.